R/recipe_run.R

```diff
@@ -752,7 +752,7 @@ recipe_copy_readme <- function(info, src) {
 
     ## now check if README is an artefact
     artefact_files <- unlist(info$artefacts[, "filenames"], use.names = FALSE)
-    if (any(grepl("README.md", artefact_files, ignore.case = TRUE))) {
+    if (any(grepl("^(?i)readme(|.md)$", artefact_files, ignore.case = TRUE))) {
```

Comment:
what is the `(?i)` doing here?

Reply:
my understanding of regex is that this should make it case insensitive so we get README and readme.

Reply:
ok that's new to me - could you use the `ignore.case = TRUE` argument instead to be slightly less cryptic?